### PR TITLE
[firebase_auth] Make providerId 'const String' for easier use in switch statements

### PR DESCRIPTION
**Pull request by dotdoom**

### CODE REVIEW

**Comment by dotdoom** [comment]
+cc @collinjackson as the original author, PTAL?

**Comment by collinjackson** [comment]
Looks good, can you update the pubspec.yaml and CHANGELOG.md for release?

**Comment by dotdoom** [comment]
Done!

File: packages/firebase_auth/pubspec.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ description: Flutter plugin for Firebase Auth, enabling Android and iOS
   like Google, Facebook and Twitter.
 author: Flutter Team <flutter-dev@googlegroups.com>
 homepage: https://github.com/flutter/plugins/tree/master/packages/firebase_auth
-version: "0.8.2"
+version: "0.8.2+1"
```
Comment (collinjackson) [inline packages/firebase_auth/pubspec.yaml]: IMO this change merits an update to 0.8.3 since the public API changed (albeit in a backwards compatible way)

**Pull request merged**